src/widgets/settingspages/ModerationPage.cpp

```diff
@@ -140,6 +156,22 @@ ModerationPage::ModerationPage()
                              }));
                          });
 
+        EditableModelView *view =
+            logs.emplace<EditableModelView>(
+                    (new ChannelLoggingModel(nullptr))
+                        ->initialized(&getSettings()->loggedChannels))
+                .getElement();
+
+        view->setTitles({"Twitch channels", "Enable Logs"});
```

Comment:
Do we really need a checkbox for "Enable logs" here? I'd prefer if this was just a list, and if a channel is in the list, that means it's being logged.
I can't think of any use-cases where you'd need to disable logging for many channels, then enable it again briefly afterwards. If you just want to stop logging a channel for a while, it would make more sense in my opinion to just remove it.

Reply:
The "Enable logs" toggle was inspired by the discussion in the issue on having adding other toggleable channel specific settings in a table as well but I guess that wouldn't be on this page anyway. Removed the toggle.